### DISCUSSION TITLE
Is there a way to customize rules for layout items positioning?

### DISCUSSION THREAD

**Discussion by pickledme** [react-grid-layout/react-grid-layout#1885] [Q&A]
Hey 👋, I'd like to achieve the behavior described in the image below.

Imagine a user models a brick wall and you want to prevent users from stacking bricks above each other (prevent bricks stack bond and allow something like running bond brick pattern)

I didn't find a way to customize positioning / deny certain positions of layout items. Can I achieve this somehow with this lib or there is something better for this specific use case?

Thanks.

![photo_2023-05-27_18-18-02](https://github.com/react-grid-layout/react-grid-layout/assets/127624613/d7ebaa11-d314-41a2-8048-b7c62175c170)

**Comment by taziksh** [comment]
Any updates on this?

**Reply by farrukh-develo** [reply]
You can use Swapy (https://swapy.tahazsh.com/)

**Comment by shipurjan** [comment]
same question here...

**Reply by farrukh-develo** [reply]
You can use Swapy (https://swapy.tahazsh.com/)

**Reply by xaronnn** [reply]
Swapy is not advanced.
